### ISSUE TITLE
railway=construction is more prominent than railway=subway

### ISSUE THREAD

**Issue by daganzdaanda** [gravitystorm/openstreetmap-carto#2317]
At this node https://www.openstreetmap.org/node/1712550942 a way with
railway=construction 
construction=subway 
splits away from a way with
railway=subway

![cph-metro](https://cloud.githubusercontent.com/assets/7693297/18096135/83cf522a-6ed9-11e6-818c-911339319d5b.png)

The subway under construction is more visible than the finished one (which is mistagged, it's not  finished in reality...)

This is a very minor issue, imho. 
But could we use the construction=* key to make a difference between the type of railway that is being constructed? We do make that difference with highways - and there it does make sense, while it could be too much for railways I believe.

So maybe it would suffice to make all railway=construction the same grey as railway=subway or even lighter? Then above-ground railway construction would be less visible, but that's OK for me for this style.

**Comment by kocio-pl** [comment]
> But could we use the construction=* key to make a difference between the type of railway that is being constructed?

Of course, we could, but we need a proposition how to visualize it and someone to prepare the code. Do you like to try it?
